modules/custom/d_demo/d_demo.install

```diff
@@ -59,5 +63,9 @@ function d_demo_uninstall() {
   if ($menu) {
     $menu->delete();
   }
+
+  /** @var \Drupal\d_demo\Service\ThemeLogoService $themeLogoService */
+  $themeLogoService = \Drupal::service('d_demo.theme_logo');
+  $themeLogoService->unsetThemeDemoLogo();
```

Comment:
Pytanie czy uninstall ma tu sens - bo jeśli ktoś sobie zmieni logo demowe na inne to po odinstalowaniu dema to logo mu się usunie. Dobrze myślę?

Reply:
after slack discussion, we agreed that this logo unset might not be a good idea. This functionallity was is removed from the ThemeLogoService Class